microsoft-365/lighthouse/m365-lighthouse-deploy-standard-tenant-configurations-overview.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Overview of using baselines to deploy standard tenant configurations"
+title: "Overview of using baselines for standard tenant configurations"
```

Comment:
Baselines are used for deploying standard configurations. If we take that word out, it loses its intent.